src/Services/Filesystem/Php.php

```diff
@@ -39,12 +40,38 @@ public function store(string $path, $content): string
     {
         $content = $this->sort($content);
 
+        // Gathering attribute keys to prevent loss of translations after expand/flatten process.
+        $flatAttributes = [];
+        foreach ($content as $key => $value) {
+            if (Str::startsWith($key, "attributes.")) {
+                $flatAttributes[preg_replace("/^attributes\./", "", $key)] = $value;
```

Comment:
This option will handle a key with a list of attributes in any translation file, although it should only be for a validation file.

See a [correctNestedAttributes](https://github.com/Laravel-Lang/publisher/blob/main/src/Services/Filesystem/Php.php#L83) method in this class.